util/src/main/java/tech/pegasys/artemis/util/mikuli/Signature.java

```diff
@@ -104,17 +115,28 @@ public static Signature random(int entropy) {
    * @param signature the signature to be copied
    */
   Signature(Signature signature) {
+    this.rawData = signature.rawData;
     this.point = signature.point;
   }
 
+  private G2Point parseSignatureBytes(Bytes signatureBytes) {
+    if (signatureBytes.size() == 96) {
```

Comment:
Should we extract constants for the 96 and 192 numbers since they seem to be repeating a bunch?

Reply:
Done.